app/src/main/java/vn/fstyle/com/androidWheelPickerView/widget/WheelPickerView.java

```diff
@@ -34,21 +36,29 @@ public WheelPickerView(Context context, AttributeSet attrs) {
 
     public WheelPickerView(Context context, AttributeSet attrs, int defStyleAttr) {
         super(context, attrs, defStyleAttr);
-        init(context);
+        setAttributeSet(context, attrs);
+    }
+
+    private void setAttributeSet(Context context, AttributeSet attrs) {
+        View parentView = LayoutInflater.from(context).inflate(R.layout.item_picker, this);
+        TypedArray typedArray = context.obtainStyledAttributes(attrs, R.styleable.WheelPickerView);
```

Comment:
WARNING: [Recycle] This `TypedArray` should be recycled after use with `#recycle()`
 Many resources, such as TypedArrays, VelocityTrackers, etc., should be recycled (with a `recycle()` call) after use. This lint check looks for missing `recycle()` calls.